app/mailers/patient_mailer.rb

```diff
@@ -196,14 +223,21 @@ def assessment_voice(patient)
       to: Phonelib.parse(patient.primary_telephone, 'US').full_e164,
       parameters: params
     )
-    add_success_history(patient)
+    History.report_reminder(patient: patient, comment: "Sara Alert sent a report reminder to this monitoree via #{patient.preferred_contact_method}.")
+    # Always update the last contact time so the system does not try and send emails again.
+    patient.update(last_assessment_reminder_sent: DateTime.now)
   rescue Twilio::REST::RestError => e
     Rails.logger.warn e.error_message
-    add_history_failed_voice(patient)
+    History.report_reminder(patient: patient, comment: "Sara Alert failed to call monitoree at #{patient.primary_telephone}.")
   end
 
   def assessment_email(patient)
-    add_fail_history_blank_field(patient, 'email') && return if patient&.email.blank?
+    if patient&.primary_telephone.blank?
```

Comment:
Think this should be` if patient&.email.blank?`

Reply:
I will triple check